packages/strapi-connector-mongoose/lib/queries.js

```diff
@@ -563,8 +565,14 @@ module.exports = ({ model, modelKey, strapi }) => {
   }
 
   function countSearch(params) {
+    // Convert `params` object to filters compatible with Mongo.
+    const filters = modelUtils.convertParams(modelKey, params);
```

Comment:
Wondering if you could use the convertRestQueryParams here to instead of convertParams (will deprecate it asap)

Reply:
@JelmerV-WFC let me know if you need some pointers ;)

Reply:
@alexandrebodin would be nice! Did not find the time yet to dive into it.

Reply:
Woops, wrong Github account :)

Reply:
Let me know when you have time on slack ;)